api/config/query.js

```diff
@@ -0,0 +1,21 @@
+ const pool = require('./pool') 
+ 
+ module.exports = {
+   /**
+    * DB Query
+    * @param {object} req
+    * @param {object} res
+    * @returns {object} object
+    */
+   query(quertText, params) {
+     return new Promise((resolve, reject) => {
```

Comment:
Why was it necessary to wrap the response of this function in a promise?

Reply:
The promise helps to query the database, the method returns a promise in the controller... I got it from the tutorial I read online but the end-points works just fine without the query.js